### PR TITLE
[WIP] e2e timezone tests

### PR DESCRIPTION
**Pull request by paulrosenzweig**
This PR branches of #11090 and adds e2e tests.

There's only one test run per timezone pair. It loads a question from the fixture data and ensures the ticks and y values are correct.

### CODE REVIEW

File: src/metabase/driver/h2.clj [hunk 1]
```diff
@@ -168,68 +168,69 @@
...
+    :OTHER                               :type/*
+    :RAW                                 :type/*
+    :REAL                                :type/Float
+    :SIGNED                              :type/Integer
+    :SMALLDATETIME                       :type/DateTime
+    :SMALLINT                            :type/Integer
+    :TEXT                                :type/Text
+    :TIME                                :type/Time
+    :TIMESTAMP                           :type/DateTime
+    (keyword "TIMESTAMP WITH TIME ZONE") :type/DateTime
```
Comment (paulrosenzweig) [inline src/metabase/driver/h2.clj]: @camsaul I added this mapping. Is it correct or should it be `:type/DateTimeWithTz`?

File: bin/build-for-test [hunk 2]
```diff
@@ -7,7 +7,7 @@ VERSION_PROPERTY_NAME="src_hash"
 source-hash() {
   # hash all the files that might change a backend-only uberjar build (for integration tests)
   (
-    find src project.clj resources/sample-dataset.db.mv.db -type f -print0 | xargs -0 shasum ;
+    find src project.clj resources/*.db.mv.db -type f -print0 | xargs -0 shasum ;
```
Comment (paulrosenzweig) [inline bin/build-for-test]: I added another h2 db in resources for TZ data.

File: src/metabase/driver/h2.clj (hunk 1 above)
Comment (camsaul) [reply src/metabase/driver/h2.clj]: It should be `:type/DateTimeWithTZ` (capital `Z`)

**Comment by mazameli** [comment]
Is this currently being worked on, and does it need a review from me?

**Comment by paulrosenzweig** [comment]
This is on pause. Once #11746 is merged, I'll update this to use that kind of snapshots.

**Comment by paulrosenzweig** [comment]
This is about a year old, so I'll close it. It might be worth resuscitating some day, but I'm not totally convinced. It's nice to have verification that the BE/FE are correctly communicating timezone. The cost of that is another Cypress test suite that just runs a few tests.

**Pull request closed without merging**